package.json

```diff
@@ -52,10 +51,12 @@
     ]
   },
   "devDependencies": {
+    "@testing-library/react": "^13.3.0",
     "eslint": "^8.14.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.29.4",
+    "jest": "^28.1.3",
```

Comment:
I don't think it is necessary since you have `"@testing-library/jest-dom"`

Reply:
I uninstalled *jest* but my only test seems to not working any more...

Reply:
It is ok, I had to add *jest-dom* to eslint plugins